gatsby/content/blog/2022/11/2022-11-22-synapse-1.72.mdx

```diff
@@ -0,0 +1,81 @@
+
+### Legacy Prometheus metric names disabled and to be removed
+
+In previous versions of Synapse, some Prometheus metrics were emitted under two different names,
+an older name that was non-compliant with OpenMetrics and Prometheus conventions and a new compliant name.
+
+Synapse v1.71 have the old metric names switched off by default.
+
+For now it's still possible to get them by using `enable_legacy_metrics: true`,
+however if you still use them you should really migrate because it will be removed in v1.73.
+
+You can find the full list of renamed metrics [here](https://matrix-org.github.io/synapse/v1.69/metrics-howto.html#renaming-of-metrics--deprecation-of-old-names-in-12).
+
+### Changes to the events received by application services (interest)
+
+[MSC3905](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3905-appservice-only-interested-in-local-users.md) has been implemented,
+Synapse now only considers local users to be interesting. In other words, the users namespace regex is only be applied against local users of the homeserver.
+
+Please note, this probably doesn't affect the expected behavior of your application service,
+since an interesting local user in a room still means all messages in the room (from local or remote users) will still be considered interesting.
+
+You can find a bit more info in the MSC and in the (upgrade notes)[https://matrix-org.github.io/synapse/v1.71/upgrade.html#changes-to-the-events-received-by-application-services-interest].
+
+## The new stuff
+
+### Threads, threads, threads!
+
+Several MSCs related to threads got implemented:
+- [MSC3856](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3856-threads-list-api.md) providing an API to fetch threads and related metadata.
+- [MSC3771](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3771-read-receipts-for-threads.md) and [MSC3773](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3773-notifications-for-threads.md) implementing per thread read receipts and per thread notification counts.
+- [MSC3874](https://github.com/matrix-org/matrix-spec-proposals/blob/justjanne/messages-endpoint-threads-filter/proposals/3874-messages-endpoint-threads-filter.md) allows to filter out messages belonging to threads from the main timeline (still considered experimental). Along with MSC3856, this should noticeably improve performance of rooms that use threads heavily.
+
+This should significantly improve user experience related to threads, being through behavior or performance impact.
+
+### Bounding events together
+
+Relationships are great, even more between events than humans!
+
+[MSC3912](https://github.com/matrix-org/matrix-spec-proposals/blob/babolivier/relation_redaction/proposals/3912-relation-based-redaction.md) gives us an easy way to redact an event with its relations of a certain type (experimental support for now).
```

Comment:
I'm going to rewrite this paragraph, because the implementation is currently very incomplete so I don't think it should be given that much attention.